grants/fondant/data.md

```diff
@@ -7,12 +7,12 @@ Grant-Link: "https://portal.devxdao.com/app/proposal/793",
 Website: "https://fondant.io/",
 Grant-Size: "Medium",
 Total-Milestones: "5",
-Current-Milestone: "M3",
-M1: "05.06.2023","Complete",
-M2: "14.09.2023","In review",
-M3: "14.10.2023","In review",
-M4: "21.11.2023","In progress",
-M5: "21.12.2023","To Do",
+Current-Milestone: "M4",
+M1: "05.06.2023","Completed",
+M2: "14.09.2023","Completed",
+M3: "21.10.2023","Completed",
```

Comment:
Hi @yetson0 ,
Please check with @Anna-defdone  for the revised date.

Reply:
changed.